frontend/static/guide/recorder/schema-migration.json

```diff
@@ -0,0 +1,101 @@
+{
+  "title": "schema-migration",
+  "steps": [
+    {
+      "type": "setViewport",
+      "width": 935,
+      "height": 833,
+      "deviceScaleFactor": 1,
+      "isMobile": false,
+      "hasTouch": false,
+      "isLandscape": false
+    },
+    {
+      "type": "navigate",
+      "url": "http://localhost:3000/",
```

Comment:
Should these still be `localhost` after releasing?

Reply:
This file is generated by Chrome Recorder, so it shouldn't be changed. We can cover it by setting a `url` in another JSON file.